bin/blueoak-server.js

```diff
@@ -3,10 +3,21 @@
  * Copyright (c) 2015-2016 PointSource, LLC.
  * MIT Licensed
  */
+var parseArgs = require('minimist');
 var server = require('../');
 
+// parse arguments
+var argv = parseArgs(process.argv.slice(2));
+
+// convert mocks from CSV into an array
+var mocks = argv.mocks || argv.m;
+if (mocks) {
+    mocks = mocks.split(',');
+}
+
 server.init({
-    appDir: process.cwd()
+    appDir: process.cwd(),
+    mocks: mocks
```

Comment:
I could add a config to define the location of mocks, but I initially chose not to because BOS appears to favor convention over configuration for the locations of other things like handlers, services, etc.

I avoided declaring mocks in the configuration for the following reasons:
1. To keep the config under test as similar as possible to the release/runtime config.
1. To be able to easily control which services are mocked without changing config (i.e.: mocks won't be loaded simply because they exist or are declared, but only if they're also identified in the CLI argument). Although, this approach could still apply if mocks are declared in config.

Let me know how you'd like to proceed here and I'll make the necessary updates.